### ISSUE TITLE
Cancel doesn't close sheet

### ISSUE THREAD

**Issue by tcurdt** [sberrevoets/SDCAlertView#161]
This doesn't work as expected:

    let alert = AlertController(title: "Title", message: "Message", preferredStyle: .ActionSheet)
    alert.addAction(AlertAction(title: "A", style: .Default) { _ in
        print("A")
    })
    alert.addAction(AlertAction(title: "B", style: .Default) { _ in
        print("B")
    })
    alert.present()

It also shows a `Cancel` button - but the button doesn't even cancel the sheet.

**Comment by tcurdt** [comment]
Turns out this works - but the `Cancel` button **has** to be `.Preferred`. That's not exactly obvious.
```
let alert = AlertController(title: "Title", message: "Message", preferredStyle: .ActionSheet)
alert.addAction(AlertAction(title: "A", style: .Default) { _ in
    print("A")
})
alert.addAction(AlertAction(title: "B", style: .Default) { _ in
    print("B")
})
alert.addAction(AlertAction(title: "Cancel", style: .Preferred))
alert.present()
```

**Comment by sberrevoets** [comment]
That's very weird indeed. Feel free to open a PR if you know what the problem is, otherwise I'll get to it "eventually".

**Comment by sberrevoets** [comment]
This is fixed in version 7.0. Thanks for reporting!
